jena-fuseki2/jena-fuseki-ui/src/styles/index.scss

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-@import 'node_modules/bootstrap/scss/bootstrap';
+@import 'node_modules/bootstrap-vue/node_modules/bootstrap/scss/bootstrap.scss';
```

Comment:
Commit message has a link to the README of Bootstrap Vue, where they changed the recommended way to import it (I got a warning using the old way)